src/Wrapper/Data.cs

```diff
@@ -517,5 +517,35 @@ public static Array Copy(Array arr)
             return new Array(ptr);
         }
         #endregion
+
+        #region Moddims
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static Array Moddims(Array arr, params int[] dims)
```

Comment:
This is more like a curious Q, is there no wrapper dim like `dim4` at the wrapper level ?

Reply:
I can't speak for the original designer of this wrapper, but I surmise that the C# Array object has rich enough features that creating a new dim4 class would be redundant. See https://docs.microsoft.com/en-us/dotnet/api/system.array?view=netcore-3.1